web/src/main/java/org/openmrs/module/web/WebModuleUtil.java

```diff
@@ -285,11 +286,11 @@ public static boolean startModule(Module mod, ServletContext servletContext, boo
 					}
 				}
 			}
-			
+
 			// mark to delete the entire module web directory on exit
 			// this will usually only be used when an improper shutdown has occurred.
-			String folderPath = realPath + "/WEB-INF/view/module/" + mod.getModuleIdAsPath();
-			File outFile = new File(folderPath.replace("/", File.separator));
+			//String folderPath = realPath + "/WEB-INF/view/module/" + mod.getModuleIdAsPath();
+			File outFile = Paths.get(realPath, "WEB-INF", "view", "module", mod.getModuleIdAsPath()).toFile();
```

Comment:
@MalshaL cant you just use

`File outFile = new File(webInf, "view", ...);`

instead of using the Paths.get() ?

Also please do not comment out code! Always remove it. We use git, if we need it back we can easily get it.